src/book/03-guides/01-main-codebase/02-script-tuner.mdx

```diff
@@ -0,0 +1,99 @@
+---
+section: Guides
+chapter: Main Codebase
+title: Running and Tuning Scripts
+description: How to tune and run scripts.
+slug: /guides/main/script-tuner
+---
+
+Scripts are static motions for the robot. They specify what joint angles to move to and how long the robot should take to go to those joint positions. For example, standing up is a very basic script telling the robot to move their joints to the stand position over one second. There can be many of these position specifications in a row to make the robot do more complex movements like getting up or kicking.
+
+The Script Engine specifies how the scripts work. It takes each joint position and time from the `.yaml` file and sends the information to the servos.
+
+Information on Script Engine is on the [Motion](/system/subsystems/motion#script-engine) page and Script Runner and Script Tuner on the [Behaviour](/system/subsystems/behaviour#script-runner) page.
+
+## Script Runner
+
+The script runner command will run a script. The command for this is
+
+```bash
+./scriptrunner Script.yaml
+```
+
+Where you would replace `Script.yaml` with the name of the script. All scripts can be found in the NUbots repository under `/module/motion/ScriptEngine/data/scripts/`.
+
+The script you are running needs to be on the robot in the `/scripts/` folder. These scripts can be robot specific. For example, if you were running `Stand.yaml` on `nugus1`, the ScriptEngine would first look for `Stand.yaml` in the `/scripts/` folder, and then it would check in the `/scripts/nugus1/` folder for a `Stand.yaml` file. If it exists, it will override with that script.
+
+## Script Tuner
+
+Script tuner is an easy way to make scripts. To use script tuner, run
+
+```bash
+./scripttuner /path/to/Script.yaml
+```
+
+Where `Script.yaml` is either the name of an existing script, if you want to edit it, or the name for a new script. `/path/to/` is the path to the script. This may be `scripts/nugus/Script.yaml`, or `scripts/nugus*/Script.yaml` where `*` is the robot number for a robot specific script.
```

Comment:
```suggestion
Where `Script.yaml` is either the name of an existing script, if you want to edit it, or the name for a new script. `/path/to/Script.yaml` is the path to the script. This may be `scripts/nugus/Script.yaml`, or `scripts/nugus*/Script.yaml` where `*` is the robot number for a robot specific script.
```